test/runtests.jl

```diff
@@ -215,14 +220,16 @@ function test_integrals(; kw...)
     end
 
     n = length(misses)
-    if n > 0 println("**** missess (n=$n) *****") end
+    if n > 0
+        println("**** missess (n=$n) *****")
+    end
     for eq in misses
         printstyled(eq, '\n'; color=:red)
     end
     return n
 end
 
-@testset "integral" begin 
-    n = test_integrals(;symbolic=false, verbose=false, homotopy=true, num_steps=2, num_trials=10)
-    @test n==11
+@testset "integral" begin
+    n = test_integrals(; symbolic=false, verbose=false, homotopy=true, num_steps=2, num_trials=10)
+    @test n == 10 
```

Comment:
if tests pass, this is really the only thing of note. note sure why one more integral works now